svelte/src/libs/translations/languages/translation to Azerbaijan

```diff
@@ -0,0 +1,101 @@
+{
+  "id": "en",
+"label": "English",
```

Comment:
These should be changed to the appropriate language

Reply:
Let's leave it, I couldn't, I couldn't do it. is there any other activity easier?

Reply:
Just give me contributor role😐
For my efforts